### DISCUSSION TITLE
Workspace enhancements

### DISCUSSION THREAD

**Discussion by djeglin** [zen-browser/desktop#2274] [Ideas]
Hi all! I've been a Firefox user for a good while now because of the ability to set up tab groups and container tabs, allowing me to have separate profiles for different clients where I need to sign in to things like O365 etc. Zen does a great job of doing this, but I think there may be a few areas where the workspaces feature could be enhanced to offer a better experience: 

1. Allow workspaces to be customised more. For instance, allow users to set up a custom icon, or even a custom letter or letters to use as their workspace icon. This will help people like me differentiate between multiple workspaces that begin with the letter A, and allow visual people to use images instead of just letters. 

2. When I used Simple Tab Groups on Firefox, one very handy feature was seeing an indicator against a group when a sound was played by a tab in that group. I don't have this in Zen at the moment, so when I hear the familiar "ping" of an O365 notification, I have to go hunting through my workspaces to find which one gave me an alert. Adding a little indicator, or even highlighting the group that made the noise, would be extremely helpful in these cases. 

3. This may be one for the mod community, but I don't know whether its something that needs some work on this end, too... Make things like toolbar themes customisable on a per-workspace level, just to allow us to have some more immediate visual feedback of which workspace we are in. At the moment I am using the super-gradient theme, and I raised a ticked there saying how much I would love to be able to use different gradients for different workspaces so I immediately know which client space I am working in. 

So far, Zen has been brilliant. I love the look, and I really feel like having something like this that isn't Chromium based is good for the community as a whole. Keep up the good work!

**Comment by mikedlr** [comment]
Number 1.  Seems to work for me

1. Right click on workspace
2. menu pops up with list of workspaces - go to right of line with your worspace click on three dots -
3. select edit workspace 
4. click on icon  - choice appears
5. click on word -  use different letter then space as work around for changing space letter.

**Reply by djeglin** [reply]
I don't seem to be able to do this any more - has this workaround been removed? I seem to now be forced into using an emoji for any new workspaces, which is not always better than just using a letter.

**Comment by virtual-orbis** [comment]
Currently, I have been using a Chromium-based browser called SIDEKICK, which, although paid, is not expensive. One of the features that I like the most about this browser is the workspaces and the applications "side" menu, as shown in the attached image. However, the functionality of workspaces is still somewhat limited in terms of cookie isolation, so that applications based on different Google/Gmail authentication accounts could be used simultaneously.
![Captura de pantalla 2024-10-27 182707](https://github.com/user-attachments/assets/da6d6eb7-a052-4cf7-b41f-821ae1d0069d)
